documentation/examples/rbac-setup.yml

```diff
@@ -0,0 +1,34 @@
+apiVersion: rbac.authorization.k8s.io/v1beta1
+kind: ClusterRole
+metadata:
+  name: prometheus
+rules:
+- apiGroups: [""]
+  resources:
+  - nodes
+  - nodes/proxy
+  - services
+  - endpoints
+  - pods
+  verbs: ["get", "list", "watch"]
+- nonResourceURLs: ["/metrics"]
+  verbs: ["get"]
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: prometheus
+  namespace: default
```

Comment:
Right, but `ClusterRoleBinding` != `RoleBinding`. See the RBAC section starting from https://kubernetes.io/docs/admin/authorization/rbac/#role-and-clusterrole.

The `Role` and `RoleBinding` resources are tied to a namespace, while the `ClusterRole` and `ClusterRoleBinding` resources allow access to cluster-wide resources across namespaces.

E.g.:

"A `ClusterRole` can be used to grant the same permissions as a `Role`, but because they are cluster-scoped, they can also be used to grant access to: [...] namespaced resources (like pods) across all namespaces (needed to run kubectl get pods --all-namespaces, for example)"

and:

"Permissions can be granted within a namespace with a `RoleBinding`, or cluster-wide with a `ClusterRoleBinding`."